tests/conftest.py

```diff
@@ -11,8 +11,16 @@
 
 
 @pytest.fixture(autouse=True)
-def debug(caplog):
-    with caplog.at_level("DEBUG", logger="dvc"):
+def reset_loglevel(request, caplog):
```

Comment:
Should this be used during the test debugging or something? Could you elaborate?

Reply:
What do you mean by test debugging? The goal of this PR is to make less output from tests by default in accordance with #1888, to make it easier to see a stack trace or a list of tests failed. 

It is still bloated to my liking, but it would be counter-intuitive to not show output at all. So I left the default level by default.

Reply:
Ah, I didn't notice the `autouse=True`. Now I see that it is indeed used. Thanks.